website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md

```diff
@@ -0,0 +1,183 @@
+---
+slug: run-mongodb-commands-ferretdb-pgedge-postgres-backend
+title: 'Run MongoDB workloads on FerretDB with pgEdge Postgres Platform as backend'
+authors: [alex]
+description: >
+  Learn how to run MongoDB workloads in FerretDB with a fully distributed pgEdge PostgreSQL as the backend.
+image: /img/blog/ferretdb-pgedge.jpg
+tags: [compatible applications, tutorial, cloud, postgresql tools, open source]
+---
+
+![Run MongoDB commands on FerretDB with pgEdge Postgres as backend](/img/blog/ferretdb-pgedge.jpg)
+
+FerretDB lets you run MongoDB workloads in open source using PostgreSQL as the backend.
+It does this by converting BSON documents to JSONB in PostgreSQL.
+
+<!--truncate-->
+
+As such, you can leverage a fully managed, distributed PostgreSQL service like [pgEdge](https://www.pgedge.com/) as your database storage.
+That will help you take advantage of its fast response time, high-availability, multi-cloud deployment options, and low data latency.
+
+In this blog post we'll dive into a complete setup on how to run FerretDB with a fully distributed pgEdge PostgreSQL as the backend.
+
+## Prerequisite
+
+Before you start, ensure you have the following:
+
+- [pgEdge account](https://www.pgedge.com/)
+- [`mongosh`](https://www.mongodb.com/docs/mongodb-shell/)
+- [Docker](https://www.docker.com/)
+
+## Setting up Postgres with pgEdge
+
+Before setting up a FerretDB instance, you need a Postgres database configured and ready.
+Then you'll pass its connection string as the `FERRETDB_POSTGRES_URI`/ `--postgres-uri flag` when you run the FerretDB instance.
+
+Start by setting up a Postgres database named `ferretdb` on pgEdge, and wait a few seconds for the database to be provisioned.
+
+![image of postgres](/img/blog/pgedge-connection.png)
+
+Once it's ready, get the connection URI credentials for the `admin` user and use it as the FERRETDB_POSTGRES_URI/ –-postgres-uri flag.
+
+## Connect FerretDB to Postgres using Docker
+
+Ensure you have Docker running.
+Run the FerretDB container using the following command without the connection uri from pgEdge configured to connect via the `FERRETDB_POSTGRESQL_URL`.
```

Comment:
I think this sentence can be rephrased. Something like this? :thinking: 

Run the FerretDB container using the following command to connect via the `FERRETDB_POSTGRESQL_URL`. Not that `FERRETDB_POSTGRESQL_URL` is not the same as `pgEdge`'s connection string.